File: eCakeShop.Services/Database/Novost.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCakeShop.Services.Database
{
    public class Novost
    {
        public int NovostID { get; set; }
        public string? Naslov { get; set; }
        public string? Sadrzaj { get; set; }
        public byte[]? Thumbnail { get; set; }
        public DateTime DatumKreiranja { get; set; }
        public int KorisnikID { get; set; }
        public virtual Korisnik Korisnik { get; set; } = null!;
    }
}

File: e_cakeshop/lib/models/novost.dart
import 'package:e_cakeshop/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';
part 'novost.g.dart';

@JsonSerializable()
class Novost {
  int? novostID;
  String? naslov;
  String? sadrzaj;
  String? thumbnail;
  String? opis;
  DateTime? datumKreiranja;
  int? korisnikID;
  Korisnik? korisnik;

  Novost();

  factory Novost.fromJson(Map<String, dynamic> json) => _$NovostFromJson(json);

  Map<String, dynamic> toJson() => _$NovostToJson(this);
}
